tests/testdata/xdc_clusters.yaml

```diff
@@ -9,6 +9,14 @@
   historyconfig:
     numhistoryshards: 1
     numhistoryhosts: 1
+  esconfig:
```

Comment:
not sure yet what all these `tests/testdata/` files are used for tbh, but the changes look good to me

Reply:
Some functional tests load specific configs to enable/disable features to test.